pkg/nodeca/nodeca.go

```
@@ -0,0 +1,181 @@
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+// removeDoubleDots replaces the last occurence of a double dot ("..") with
```

golangci-lint (2.11.4)

[error] `occurence` is a misspelling of `occurrence`

(misspell)